packages/integration-tests/tests/api/addresses.test.ts

```diff
@@ -0,0 +1,227 @@
+import { localConfig } from "../../src/config";
+import { Buffer, Token, Wallets } from "../../src/entities";
+import { Helper } from "../../src/helper";
+import { Playbook } from "../../src/playbook/playbook";
+
+describe("Address", () => {
+  jest.setTimeout(localConfig.extendedTimeout);
+
+  const helper = new Helper();
+  const playbook = new Playbook();
+  const bufferFile = "src/playbook/";
+  let apiRoute: string;
+  let contract: string;
+  let token: string;
+  let txHash: string;
+  let response;
+
+  describe("/address/{address}", () => {
+    beforeAll(async () => {
+      await playbook.deployNFTtoL1();
+      await playbook.deployNFTtoL2();
+      await playbook.deployMultiCallContracts();
+      await playbook.deployMultiTransferETH();
+      await playbook.deployGreeterToL2();
+      await playbook.useMultiCallContracts();
```

Comment:
Why using contracts is here? I think what we should do is use ethers and execute testing logic within the tests instead of making a bunch of calls and then testing what happened.
The flow should be:
1) Deploy contracts in `beforeAll`
2) In every test interact with a contract and make proper checks. Interactions can be done without using hardhat scripts, we should just use SDK.